.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 100,
```

Comment:
Curious about the rational behind changing the default value of 80 (which seems a sensible default value to me)?
cf. https://prettier.io/docs/en/options.html#print-width

Reply:
_Please note that the .prettierrc files existed before this PR with the width set to 100 already. So I'm not sure who changed the setting originally._

Anyway I can try to formulate an argument. Since we are in 2023, with our shiny retina screens our favourite code editors can display at least 130 characters / line even with the sidebar open (and with a small 13" screen). 80 seems really contrived and leave of lot of unused space.

But ultimately (and just as the other thing you raised) this is really not important to me. So whatever, if we vote in the team and people are happy with using 1/3 of their screen space I'm fine with it.

Reply:
So great, we should get an agreement quickly then.
One thing sure: We can't push code modifying other developers' code without consulting them. 

If you don't want this PR to be blocked, let's override at first prettier rules for the projects that used other conventions so there is no impact at all on them and then we can review the shared rules with everyone.